fabric-resource-conditions-api-v1/src/main/java/net/fabricmc/fabric/impl/resource/conditions/ResourceConditionsImpl.java

```diff
@@ -204,4 +208,37 @@ public static boolean tagsPopulatedMatch(JsonObject object, RegistryKey<? extend
 
 		return true;
 	}
+
+	public static ConditionJsonProvider featuresEnabled(Identifier id, final FeatureFlag... features) {
+		final Set<Identifier> ids = FeatureFlags.FEATURE_MANAGER.toId(FeatureFlags.FEATURE_MANAGER.featureSetOf(features));
```

Comment:
This should be sorted to ensure that the order in which they're generated is consistent.